lms/djangoapps/lti_provider/migrations/0002_auto_20160325_0407.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import provider.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('lti_provider', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='lticonsumer',
+            name='consumer_secret',
+            field=models.CharField(default=provider.utils.short_token, unique=True, max_length=32),
```

Comment:
@e0d I've added missing migrations for `consumer_secret` field. Does this make sense to you ?